Sources/Pact.swift

```diff
@@ -14,7 +14,11 @@ public class Pact {
     self.interactions = interactions
   }
 
-  func payload() -> [String: Any] {
+  public func addInteractions(_ interaction: Interaction) {
+    self.interactions.append(interaction)
```

Comment:
What was the reason for exposing this?

Reply:
Long story short: since I need it for my test setup. 
Thinking about it I'm sure I can find a solution without needing it here